addons/spreadsheet/static/description/icon.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M46 42C46 44.2091 44.2091 46 42 46L4 46L4 8C4 5.79086 5.79086 4 8 4L46 4L46 42Z" fill="#1A6F66"/>
```

Comment:
Indentation 🙏🏼

Reply:
I compressed the svg into a single line as we did for all the other icons. 👍